main.py

```diff
@@ -351,6 +351,9 @@ def ui_access_wrapper(self, *args, **kwargs):
     get_redirect_route(
         r'%s' % feconf.TOPICS_AND_SKILLS_DASHBOARD_DATA_URL,
         topics_and_skills_dashboard.TopicsAndSkillsDashboardPageDataHandler),
+    get_redirect_route(
+        r'%s<skill_id>' % feconf.UNASSIGN_SKILL_DATA_HANDLER_URL,
```

Comment:
@Showtim3, Can you confirm that this URL is ready for client/public to use?

Reply:
Hey @DubeySandeep how should i confirm this?

Reply:
Both the screenshots that i have added are fetching data from this URL.